sonar-db/src/main/resources/org/sonar/db/version/schema-h2.ddl

```diff
@@ -54,6 +54,7 @@ CREATE TABLE "GROUPS" (
 
 CREATE TABLE "SNAPSHOTS" (
   "ID" INTEGER NOT NULL GENERATED BY DEFAULT AS IDENTITY (START WITH 1, INCREMENT BY 1),
+  "UUID" VARCHAR(50) NOT NULL,
```

Comment:
why not create a unique index on this column? I'm pretty sure we need it for the future changes of SQL request but that it will also be used in the next migrations